conjure_python_client/_serde/decoder.py

```diff
@@ -62,6 +63,8 @@ def check_null_field(
             cls, obj, deserialized, python_arg_name, field_definition):
         if isinstance(field_definition.field_type, ListType):
             deserialized[python_arg_name] = []
+        elif isinstance(field_definition.field_type, SetType):
+            deserialized[python_arg_name] = []
```

Comment:
should this not be a frozenset?

Reply:
i dont think so?

Reply:
it should. otherwise the actual type of a set member in an object will vary depending on the wire format of the response.
- The type will be `frozenset` if the collection is present ex: `{ "foo":[1,2] }`
- The type will be `array` if the collection is absent. ex: `{}`, `{ "foo": null }`

Reply:
oh misunderstood which thing you were suggesting swapping to frozen set. you are correct

Reply:
fixed and added a test